lib/src/lock_file.dart

```diff
@@ -262,12 +249,12 @@ class LockFile {
       };
     });
 
-    var sdks = {'dart': dartSdkConstraint.toString()};
-    if (flutterSdkConstraint != null) {
-      sdks['flutter'] = flutterSdkConstraint.toString();
-    }
-
-    var data = {'sdks': sdks, 'packages': packageMap};
+    var data = {
+      'sdks': mapMap(sdkConstraints,
+          value: (_, constraint) => constraint.toString()),
+      'packages': packageMap
+    };
+    print(data);
```

Comment:
Is this intended? It doesn't look like there used to be a `print` and this is the first in the file